### PR TITLE
Add skip client to forestry configs

### PR DESCRIPTION
**Pull request by jeffsee55**

### CODE REVIEW

**Review by jamespohalloran — CHANGES_REQUESTED** [review]

File: packages/@tinacms/app/src/Playground.tsx [hunk 1]
```diff
@@ -73,7 +73,7 @@ const Playground = () => {
         <div className="graphiql-doc-explorer-content">
           <div className="graphiql-markdown-description">
             {noAutoQueries
-              ? 'No autoqueries found, the Tina dev or build command was likely run with the "--noSDK" option. Learn about how to generate autoqeries.'
+              ? 'No autoqueries found, the Tina config is likey set to client.skip = true. Learn about how to generate autoqeries.'
```
[the remark this answers is not in the document]
Comment (jeffsee55) [reply packages/@tinacms/app/src/Playground.tsx:76]:
```suggestion
              ? 'No auto-generated queries found, the Tina config is likely set to client.skip = true'
```

[the remark this answers is not in the document]
Comment (jeffsee55) [reply packages/@tinacms/app/src/Playground.tsx:76]: Yeah sorry I thought I fixed that. The link is included in the `Learn more here` tag below.

File: packages/@tinacms/app/src/Playground.tsx [hunk 2]
```diff
@@ -73,7 +73,7 @@ const Playground = () => {
         <div className="graphiql-doc-explorer-content">
           <div className="graphiql-markdown-description">
             {noAutoQueries
-              ? 'No autoqueries found, the Tina dev or build command was likely run with the "--noSDK" option. Learn about how to generate autoqeries.'
+              ? 'No auto-generated queries found, the Tina config is likely set to client.skip = true'
               : "Tina's autogenerated queries can be found here as well as any queries you may have defined yourself."}{' '}
```
Comment (jeffsee55) [inline packages/@tinacms/app/src/Playground.tsx:77]:
```suggestion
              : "Tina's auto-generated queries can be found here as well as any queries you may have defined yourself."}{' '}
```

**Comment by logan-anderson** [comment]
Looks good to me other then that small comment I had. Looks like there is some conflicts with main but other then that looks good.

**Pull request merged**